mirgecom/diffusion.py

```diff
@@ -100,14 +104,16 @@ class DiffusionBoundary(metaclass=abc.ABCMeta):
 
     @abc.abstractmethod
     def get_grad_flux(
-            self, discr, dd, u, *, quadrature_tag=DISCR_TAG_BASE):
-        """Compute the flux for grad(u) on the boundary corresponding to *dd*."""
+            self, discr, dd_vol, dd_bdry, u, *,
```

Comment:
Why not construct `dd_vol` from `dd_bdry`? (Here and below.)

Reply:
Fixed.